ui/src/AzureOutputChannel.ts

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { OutputChannel, ViewColumn } from "vscode";
+
+export class AzureOutputChannel implements OutputChannel {
```

Comment:
Can we name this `AzExtOutputChannel`? I want to start following the naming pattern I used for `AzExtTreeItem` vs. `AzureTreeItem`. "AzExt" should be used for anything that could apply outside of Azure (e.g. an attached mongo database) and "Azure" should be reserved for things that _only_ work in Azure.